apps/website/src/app/learn/page.tsx

```diff
@@ -0,0 +1,292 @@
+                </Text>
+                <Link href="https://pse.dev/resources" isExternal>
+                    <Text
+                        borderBottomWidth={"1px"}
+                        borderBottomColor={"white"}
+                        _hover={{ borderBottomColor: "transparent" }}
+                        fontSize="20px"
+                        fontWeight="normal"
+                    >
+                        Learn more
+                    </Text>
+                </Link>
+            </VStack>
+            <VStack mt={"40px"}>
+                <VStack>
+                    <Text fontSize={"30px"} fontWeight={"500"}>
+                        Characteristics
+                    </Text>
+                    <InfoCard texts={infoCardTexts[0]} />
+                </VStack>
+            </VStack>
+        </VStack>
+    )
+
+    return (
+        <VStack>
+            <VStack h="1187px">
+                <Image
+                    src="https://semaphore.cedoor.dev/guy-shadow-horizontal.png"
+                    width={0}
+                    height={0}
+                    sizes="100vw"
+                    style={{ width: "auto", height: "1187px", zIndex: "-1", position: "absolute" }}
+                    alt="Shadow of a person"
+                />
+                <Tabs position="relative" variant="unstyled" align={"center"} mt={"170px"}>
+                    <TabList gap={"40px"}>
+                        <Tab px={0} fontSize={"24px"}>
+                            About Semaphore
+                        </Tab>
+                        <Tab px={0} fontSize={"24px"}>
+                            About Zero Knowledge
+                        </Tab>
+                    </TabList>
+                    <TabIndicator mt="-1.5px" height="2px" bg="white" borderRadius="1px" />
+                    <TabPanels mt={"80px"}>
+                        <TabPanel>{renderTabBlockSemaphore()}</TabPanel>
+                        <TabPanel>{renderTabBlockZeroKnowledge()}</TabPanel>
+                    </TabPanels>
+                </Tabs>
+            </VStack>
+            <VStack p={"128px 80px"}>
+                {sectionBlockTexts.map((sectionBlockText, i) => (
+                    <VStack key={i}>
+                        <SectionBlock
+                            title={sectionBlockText.title}
+                            description={sectionBlockText.description}
+                            linkText={sectionBlockText.linkText}
+                            linkUrl={sectionBlockText.linkUrl}
+                            codeText={sectionBlockText.codeText}
+                            itemList={sectionBlockText.itemList}
+                        />
+                        {i !== sectionBlockTexts.length - 1 && <Divider my={"68px"} borderColor={"alabaster.600"} />}
+                    </VStack>
+                ))}
+            </VStack>
+            <VStack backgroundColor={"darkBlue"} w={"100vw"}>
```

Comment:
Both these Carousel sections are missing the title I think.

Reply:
Yes, true